app_dart/lib/src/model/appengine/key_helper.dart

```diff
@@ -157,9 +157,9 @@ class KeyHelper {
         }
         if (key.id != null) {
           if (key.id is String) {
-            element.name = key.id;
+            element.name = key.id as String;
           } else if (key.id is int) {
-            element.id = Int64(key.id);
+            element.id = Int64(key.id as int);
           }
         }
```

Comment:
You can avoid the 2 casts by extracting a `id` local variable.

Reply:
Done and done, thanks

Reply:
Actually you can also remove the surrounding if-not-null

Reply:
That thought occurred to me and then for some reason I didn't do it.  Thanks!